api/libproject/devstart.rb

```diff
@@ -730,6 +730,14 @@ def with_cloud_proxy_and_db_env(cmd_name, args)
 end
 
 def circle_deploy(cmd_name, args)
+  if ENV.has_key("CIRCLE_BRANCH") and ENV.has_key("CIRCLE_TAG")
```

Comment:
Can we say in a comment when CIRCLE_BRANCH and CIRCLE_TAG are passed in?

Reply:
Linked to the relevant docs which explain this, also logged their values here.